lib/administrate/field/time.rb

```diff
@@ -0,0 +1,11 @@
+require_relative "base"
+
+module Administrate
+  module Field
+    class Time < Base
+      def default
+        data && data.strftime("%H:%M:%S")
```

Comment:
Exactly, although you may want to choose a name other than `time`. Perhaps `military` or `military_with_seconds` or something along those lines?